F50/Preferences/Preferences.android.cs

```diff
@@ -6,6 +6,10 @@ namespace Xamarin.F50
 {
 	public partial class Preferences
 	{
+		public Preferences(global::Android.Content.Context context)
+		{
+		}
```

Comment:
An example of a platform-specific overload.

Reply:
Can we leave this bit out of this PR?  We may add these sorts of things for API's at some point, but I don't think Preferences will ever have it, as we simply need application context for it.

Reply:
OK, this was just an example.